### DISCUSSION TITLE
There is no way to test the email notifications through the Azure Portal

### DISCUSSION THREAD

**Discussion by VWACBrick** [Azure/api-management-developer-portal#2550] [Feature request]
## Bug description

The [documentation for API Management](https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-configure-notifications) indicates "sending a test", but we can't see anywhere in the Azure Portal UI a way to trigger a test email to validate the email template. 

## Reproduction steps

1. Go to the Azure API Portal
2. Click on "Notification templates" on the left
3. Select a template

## Expected behavior

We should see an option for "Send Test Email". Maybe show your current email your using for Azure, but allow it to be changed to something else if you trying to test Gmail or Outlook layouts.

**Comment by azaslonov** [comment]
Hi @VWACBrick, indeed, we had "Send test email" functionality in the old publisher portal, probably we missed it when moving to Azure portal.

**Comment by pacharanero** [comment]
It is highly frustrating that there is no way to preview or test the emails that are going to be sent to customers. This is basic stuff. This kind of lack of functionality makes it increasingly hard to run a professional-looking API service using Azure APIM.

**Comment by rsch136** [comment]
Hi,
Is there an ETA on this feature request?
